images/runtime/php-fpm/8.1/base.bullseye.Dockerfile

```diff
@@ -76,10 +76,7 @@ RUN docker-php-ext-configure pdo_odbc --with-pdo-odbc=unixODBC,/usr \
         xsl
 RUN pecl install redis && docker-php-ext-enable redis
 # https://github.com/Imagick/imagick/issues/331
-RUN set -eux; \
-    if [[ $PHP_VERSION != 8.* ]]; then \
-        pecl install imagick && docker-php-ext-enable imagick; \
-    fi
+RUN pecl install imagick && docker-php-ext-enable imagick
 
```

Comment:
Naive question: do you know why we don't have a similar change for PHP 8.0?

Reply:
So I think this is just because Cormac regenerated the bullseye images. PHP-FPM 8.0 is buster based

Reply:
Thanks!